sdk/tests/js/tests/tex-image-and-sub-image-with-image-bitmap-utils.js

```diff
@@ -53,10 +53,10 @@ function runOneIterationImageBitmapTest(useTexSubImage2D, bindingTarget, program
     }
 
     if (optionsVal.is3D)
-        debug('Testing texSubImage3D' + ' with flipY=' + flipY + ' and premultiplyAlpha=' + premultiplyAlpha +
+        console.log('Testing texSubImage3D' + ' with flipY=' + flipY + ' and premultiplyAlpha=' + premultiplyAlpha +
                 ', bindingTarget=' + (bindingTarget == gl.TEXTURE_3D ? 'TEXTURE_3D' : 'TEXTURE_2D_ARRAY'));
     else
-        debug('Testing ' + (useTexSubImage2D ? 'texSubImage2D' : 'texImage2D') +
+        console.log('Testing ' + (useTexSubImage2D ? 'texSubImage2D' : 'texImage2D') +
```

Comment:
This will change the output of the test, moving it from the web page to the console and making it more difficult to interactively tell which part of the test is running. It would be better to compute the string:

var str = 'Testing ' + ...

then:
debug(str);
console.log(str);

Could you make this change in a follow-on pull request?